### PR TITLE
add: script to install fresh set of dependencies

### PR DESCRIPTION
**Pull request by NBNARADHYA**
Install a fresh set of dependencies in all directories

Now just run `npm run dep:install`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,11 @@
     "lint": "cd server && npm run lint && cd ../frontend && npm run lint",
     "lint:fix": "cd server && npm run lint:fix && cd ../frontend && npm run lint:fix",
     "server": "cd server && node index.js",
-    "dev": "cd frontend && npm run dev"
+    "dev": "cd frontend && npm run dev",
+    "dep:install:root": "rm -rf node_modules && npm install",
+    "dep:install:frontend": "cd frontend && rm -rf node_modules && npm install && cd ..",
+    "dep:install:server": "cd server && rm -rf node_modules && npm install && cd ..",
+    "dep:install": "npm run dep:install:root && npm run dep:install:frontend && dep:install:server"
```
Comment (ridhishjain) [inline package.json:15]: `"dep:install": "npm run dep:install:root && npm run dep:install:frontend && npm run dep:install:server"`

Comment (NBNARADHYA) [reply package.json:15]: ?

**Review by cjchirag7 — APPROVED** [review]
LGTM! Just pull the latest changes from master

File: package.json (hunk 1 above)
Comment (ridhishjain) [reply package.json:15]: shouldn't it be `npm run dep:install:server`

Comment (ridhishjain) [inline package.json:15]: missing `npm run` in line 15

**Review by ridhishjain — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (cjchirag7) [inline package.json:15]:
```suggestion
    "dep:install": "npm run dep:install:root && npm run dep:install:frontend && npm run dep:install:server"
```
This should do

**Pull request merged**